### DISCUSSION TITLE
How to implement a framebuffer on moddable application?

### DISCUSSION THREAD

**Discussion by sv-ei** [Moddable-OpenSource/moddable#868] [Q&A]
when the application completely uses Piu

**Comment by phoddie** [comment]
There is insufficient information here to understand the question. Please provide a complete question.

**Comment by sv-ei** [comment]
If I have to capture the information/Data on the screen and create a frame buffer is there a way that I could use to capture the data especially when my application uses Piu to draw content rather than poco rendering. 

An example would be to capture the screens of Image example under Piu at a given point in time.
